### DISCUSSION TITLE
Balance sheet report needs selectable precision (rounding to the nearest integer amount)

### DISCUSSION THREAD

**Discussion by ylavoie** [ledgersmb/LedgerSMB#5854] [Ideas]
There should be a way to select the precision to use for amounts on the report.
Currently, we use the preference precision, but this should be overrideable by report. Same for Income.

**Comment by ehuelsmann** [comment]
I'm not sure what the subject has to do with the description. Could you please bring them inline?

Also: I don't quite agree: I think we have a precision which needs to be applied to currency amounts everywhere, with the sole exception of parts prices, which should allow for higher precision for registration of unit prices on very-low-price units.

**Comment by ehuelsmann** [comment]
Why do you say that we need to be able to set the precision? Because you ran into a case where the preference setting wasn't picked up correctly? Or because you have a general need to report 5-digit currency amounts on your balance sheet while your preference is set to 2-digit precision?

**Comment by ylavoie** [comment]
Accountants in Canada use figures rounded to the dollar, because business income tax reports are expecting dollars only, no cents, while all transactions are using amounts with 2 digits after the point. So we should be able to produce those reports with the desired precision.

**Reply by ehuelsmann** [reply]
I think you need more than just rounding: you want the numbers to add up correctly and result in the same numbers on assets as on liabilities (making sure the balance still balances).
